python/pyarrow/tests/test_csv.py

```diff
@@ -332,11 +332,13 @@ def read_bytes(self, b, **kwargs):
         :param kwargs: arguments passed on to open the csv file
         :return: b parsed as a single RecordBatch
         """
+        raise NotImplementedError
 
     @property
     @abc.abstractmethod
     def use_threads(self):
-        """Return true if this test is multi-threaded"""
+        """Whether this test is multi-threaded"""
+        raise NotImplementedError
```

Comment:
The NotImplementedError shouldn't be needed since the class cannot be created if an abstract method is not implemented

Reply:
I just realized there is an `abstractproperty` decorator which I should have used here. Guess I'll fix that in my next change someday

Reply:
Hmm, you're right.